bower.json

```diff
@@ -29,7 +29,7 @@
     "cf-icons": "^1.0.0",
     "cf-pagination": "^1.0.0",
     "cf-typography": "^1.0.0",
-    "cf-layout": "^1.0.0",
+    "cf-layout": "jimmynotjim/cf-layout#new-breakpoints",
```

Comment:
Unfortunately it's not that easy, there are actual changes to cf-layout that are needed here because most of the breakpoints in cf-layout were px values and couldn't be updated through variables.